tests/pfcwd/test_pfcwd_function.py

```diff
@@ -263,6 +263,8 @@ def get_pkt_cnts(self, queue_oid, begin=True):
             begin(bool) : if the counter collection is before or after the test
 
         """
+        if self.dut.facts["asic_type"] in ['cisco-8000']:
+            return
```

Comment:
If return here, the cntr_val would not be updated? will it be failed during verify_pkt_cnts?